src/handy-scroll-proto.js

```diff
@@ -91,6 +91,10 @@ let handyScrollProto = {
                 scrollBody.getBoundingClientRect().bottom :
                 window.innerHeight || dom.html.clientHeight;
             mustHide = ((containerRect.bottom <= maxVisibleY) || (containerRect.top > maxVisibleY));
+
+            if (scrollBody) {
+                widget.style.bottom = `calc(100% - ${scrollBody.getBoundingClientRect().bottom}px`;
+            }
```

Comment:
Actually, this is a breaking change. Currently, handy-scroll [provides](https://github.com/Amphiluke/handy-scroll#special-cases) a working solution for attaching the floating scrollbar to a scrollable container rather than the viewport’s bottom edge.


https://github.com/Amphiluke/handy-scroll/assets/1937729/e67568e6-82af-4926-9634-0ccb465d699f



Unfortunately, the proposed change breaks positioning of the widget in this special case. Could you elucidate what exactly doesn’t suit you in the existing solution?

Reply:
You're right I ended up being able to achieve the same result using these special classes. I likely didn't test them properly or failed to import all of the CSS properly when I tested it earlier. I will revert